### PR TITLE
LPS-115938 

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
> /cc @marcosapmf

Thank you @carolmariaabb :D

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js [hunk 1]
```diff
@@ -64,11 +64,7 @@ class Form extends Component {
 			showPublishAlert,
 		} = this.props;
 
-		if (this.isFormBuilderView()) {
-			this.syncActiveNavItem(this.state.activeNavItem);
-		}
-
-		const {paginationMode} = this.state;
+		const {activeNavItem, paginationMode} = this.state;
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js:67]: Great job on this commit, @carolmariaabb. 😄 👏

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 2]
```diff
@@ -53,7 +53,7 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 		navigationItems="<%= ddmFormAdminDisplayContext.getFormBuilderNavigationItems() %>"
 	/>
 
-	<nav class="management-bar management-bar-light navbar navbar-expand-md toolbar-group-field" id="<portlet:namespace />managementToolbar">
+	<nav class="hide management-bar management-bar-light navbar navbar-expand-md toolbar-group-field" id="<portlet:namespace />managementToolbar">
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:56]: Good job, @carolmariaabb. 😉

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/natocesarrego/liferay-portal/pull/653. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/3c929158858d~2...3c929158858d) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**